heron/spi/src/java/BUILD

```diff
@@ -146,3 +146,19 @@ java_library(
     ),
     deps = statemgr_deps_files,
 )
+
+java_binary(
+    name = "spi-unshaded",
+    srcs = glob([
+        "**/spi/utils/**/*.java",
+        "**/spi/common/**/*.java",
+    ]),
+    deps = utils_deps_files,
+)
+
+genrule(
+    name = "heron-spi-jar",
+    srcs = [":spi-unshaded_deploy.jar"],
+    outs = ["heron-spi.jar"],
+    cmd  = "cp $< $@",
+)
```

Comment:
My thought is that this would live along side the api and storm jars and can be added as dependency the same way as those two. I think generally anything that coding against the spi interfaces will be expected to be run from the heron-cli or something like that right?